### PR TITLE
Bug 1218115 - Failure in test_settings_media_storage_change.py in camera.launch

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=45138c81b8e86e188e5b8ab059ab086920fdc701

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_media_storage_change.py [hunk 1]
```diff
@@ -35,6 +35,9 @@ def test_settings_media_storage_change(self):
         # Apparently, the ability of making a photo is very fragile on low memory devices
         self.apps.kill_all()
 
+        # Workaround for bug 1218115 where the camera doesn't launch without this small pause
+        import time
+        time.sleep(0.5)
         camera = Camera(self.marionette)
         camera.launch()
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_media_storage_change.py]: Could we increase the `launch_timeout` instead?

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_media_storage_change.py]: That's not going to work, the camera app doesn't launch at all when trying it too quickly after kill_all().
I believe the unit tests use similar workarounds for this:
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/unit/test_kill_multiple.py#19
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/unit/test_killall.py#17

Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_media_storage_change.py]: Okay. I don't see a better way :)

**Pull request merged**